.github/workflows/ci.yml

```diff
@@ -51,6 +51,7 @@ jobs:
             ${{ runner.os }}-node-${{ matrix.NODE_VERSION }}-
     - run: npm ci
     - run: npm run lint
+    - run: npm run test:types
```

Comment:
Do we need to run this for various Node and DB environments of can we move this into its a separate job `check-types`?